### DISCUSSION TITLE
Time selector doesn't update in DataRangePicker

### DISCUSSION THREAD

**Discussion by arl-lakera** [rsuite/rsuite#3617] [Q&A] [answered]
Hi all, first things first: thanks for the great work and documentation of your components.

I have a quick question regarding the `DataRangePicker` component with hour & minutes time format enabled: `MM/dd/yyyy HH:mm` . When a default time is chosen, e.g. `Today`, the header shows the proper time (green marked), however, the inner time selector (red marked) keeps the old value and do not update to e.g. `00:00` and `23:59`. 

<img width="542" alt="Screenshot 2024-02-06 at 15 14 06" src="https://github.com/rsuite/rsuite/assets/97281608/8904a3a0-a84d-437b-a89b-b2792bf39a46">

Is this behaviour expected? I'd imagine the time selector should also be updated to match the header. Otherwise, is there a callback I can use to keep the inner time selector up to date with the header?

Thanks a lot for your help!

**Comment by simonguo** [comment]
Hi @arl-lakera Can you provide your code to reproduce this issue?

**Reply by arl-lakera** [reply]
@simonguo it can be reproduced from the documentation examples: https://rsuitejs.com/components/date-range-picker/#customize-the-date-format thanks for looking into it!

**Reply by mayank-permiso** [reply]
@simonguo +1 on this issue. It can be replicated easily. Any updates on the same? 
@arl-lakera , were you able to figure this out?

**Reply by simonguo** [reply, marked as answer]
https://github.com/rsuite/rsuite/assets/1203827/25d7b8d3-c2bd-4e8d-af72-8ac082878611

Did you click the OK button after the update time?

**Reply by arl-lakera** [reply]
It seems to working as expected now, @simonguo. Closing this, thanks for your review!

**Reply by mayank-permiso** [reply]
I am sorry but my issue was, if we update the time via text, then the callender's time does not syncs up. @simonguo 
![isssue1](https://github.com/rsuite/rsuite/assets/127431296/071ca4bf-82be-4ec6-a117-5fa4749ce3be)
